ldapdb/models/base.py

```diff
@@ -91,7 +91,7 @@ def get_field_value(field, instance):
         if create:
             old = None
         else:
-            old = cls.objects.using(using).get(dn=self._saved_dn)
+            old = cls._base_manager.using(using).get(dn=self._saved_dn)
```

Comment:
This change from `cls.objects` to `cls._base_manager` is the only non-test change in the PR.